.idea/codeStyles/Project.xml

```diff
@@ -1,248 +1,30 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <option name="USE_SAME_INDENTS" value="true" />
```

Comment:
Please do not delete these files, they are here to ensure that all developers share the same codestyles

Reply:
Sorry! I didn't realize. Could you please guide me on how to proceed? Do I need to clone the repository again or create a new branch?

Reply:
you can revert this commit and then `cherrypick` your changes from the first commit.
Or if you don't want to do that just delete this branch, close this PR and open a new branch/pr after redoing the changes.

Reply:
Okay thanks!